src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days === 3) {
+    return 100;
+  }
+
+  if (days === 6) {
+    return 220;
+  }
+
+  if (days === 7) {
+    return 230;
+  }
+
+  return 80;
```

Comment:
The current implementation only checks for specific day values (3, 6, 7) and returns a fixed cost for these. For any other number of days, it defaults to returning 80. If this is not the intended behavior, you might want to implement a more dynamic calculation or add more conditions to handle other cases.